mos_tests/rabbitmq_oslo/test_restarts.py

```diff
@@ -0,0 +1,249 @@
+#    Copyright 2016 Mirantis, Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import logging
+import random
+import re
+import time
+
+import pytest
+
+from mos_tests import settings
+
+
+logger = logging.getLogger(__name__)
+
+
+def vars_config(remote):
+    """Prepare variables and different paths
+    :param remote: SSH connection point to controller.
+    """
+    config_vars = {
+        'repo': settings.RABBITOSLO_REPO,
+        'pkg': settings.RABBITOSLO_PKG,
+        'root_path': '/root/',
+        'nova_user': 'nova'}
+    # like: /root/oslo.messaging-check-tool/
+    config_vars['repo_path'] = '{0}{1}/'.format(
```

Comment:
Let's define repo_path as constant (e.g. `/root/rabbit_oslo_tool`) and later clone repository to this path (like `git clone <url> <path>`)